connectivity/builder/builder.go

```diff
@@ -73,8 +73,29 @@ var (
 var (
 	clientLabel  = map[string]string{"name": "client"}
 	client2Label = map[string]string{"name": "client2"}
+)
+
+type testBuilder interface {
+	build(ct *check.ConnectivityTest, templates map[string]string)
+}
 
-	primaryTests = []testBuilder{
+// NetworkPerformanceTests returns a slice with network performance connectivity tests.
+func NetworkPerformanceTests(ct *check.ConnectivityTest) error {
+	return getTests(ct, []testBuilder{networkPerf{}})
+}
+
+// ConnDisruptTests returns a slice with conn disrupt connectivity tests.
+func ConnDisruptTests(ct *check.ConnectivityTest) error {
+	tests := []testBuilder{
+		noInterruptedConnections{},
+		noIpsecXfrmErrors{},
+	}
+	return getTests(ct, tests)
+}
+
+// ConcurrentTests returns a slice with all connectivity tests that can be run concurrently.
+func ConcurrentTests(ct *check.ConnectivityTest) error {
```

Comment:
how did you determine which tests can run concurrently? for example, i'm not sure how `noPolicies` can run concurrently with `allowAllExceptWorld` 💭

Reply:
Concurrent tests will run in separate namespaces.
Probably I need to mention this somewhere...

Reply:
ah ok sounds good, yeah that wasn't obvious to me from this pull request.

Reply:
I've improved function description and PR comment. Thanks!